homework-1/css/style.css

```diff
@@ -0,0 +1,1220 @@
+  padding-top: 30px;
+  padding: 67px 0 14px;
+  max-width: 142px;
+  margin: 0 auto;
+}
+.header__banner-submenu {
+  padding: 0 20px;
+  position: absolute;
+  width: 100%;
+  top: 198px;
+}
+.first-item {
+  padding-right: 26px;
+  padding-top: 6px;
+}
+.submenu__item-middle {
+  padding-top: 5px;
+}
+.last-item {
+  padding-left: 24px;
+  padding-top: 6px;
+}
+.submenu__list {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+  background: #FFFFFF;
+  box-shadow: 0px 12px 24px #F2F2F2;
+  border-radius: 5px;
+  border-top: 2.5px solid #37E63E;
+}
+.submenu__list a {
+  color: #7f7f7f;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 20px;
+  text-align: center;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  transition: 0.5s;
+}
+.submenu__list a:hover {
+  color: #37e63e;
+}
+.submenu__link {
+  border-right: 1.5px solid #e5e5e5;
+  border-left: 1.5px solid #e5e5e5;
+  padding: 0 24px;
+}
+.header__banner {
+  height: 211px;
+}
+.fixed-top {
+  padding: 0 19px;
+  top: 63px;
+}
+#list-top {
+  height: 40px;
+}
+#banner {
+  height: 211px;
+}
+#banner-submenu {
+  padding: 0 19px;
+  top: 62px;
+}
+#header {
+  padding: 18px 29px;
+}
+#header ul {
+  top: -92px;
+}
+.link-hover-center a {
+  display: inline-block;
+  color: #22BF95;
+  line-height: 1;
+  text-decoration: none;
+  cursor: pointer;
+  position: relative;
+}
+.link-hover-center a:after {
+  display: block;
+  content: "";
+  background-color: #22BF95;
+  height: 1.5px;
+  width: 0%;
+  left: 50%;
+  position: absolute;
+  -webkit-transition: width 0.3s ease-in-out;
+  -moz--transition: width 0.3s ease-in-out;
+  transition: width 0.3s ease-in-out;
+  transform: translateX(-50%);
+}
+.link-hover-center.active a:after {
+  width: 50%;
+}
+.link-hover-center a:hover:after,
+.link-hover-center a:focus:after {
+  width: 50%;
+}
+.tips {
+  padding-top: 52px;
+}
+
+a.tips-active {
+  color: #37e63e;
+}
+a.link-active {
+  color: #37e63e;
+}
+.tips-sub {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.tips-list {
+  position: relative;
+  margin-top: 24px;
+}
+.tips-item-wrap {
+  display: flex;
+  align-items: center;
+}
+.tips-item__num {
+  font-family: "Actor-Regular";
+  font-size: 40px;
+  line-height: 48px;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #E5E5E5;
+  padding-left: 15px;
+  margin-bottom: 8px;
+}
+.tips-item__title {
+  font-family: "Aclonica-Regular";
+  font-size: 16px;
+  line-height: 24px;
+  color: #191919;
+  vertical-align: super;
+  padding-left: 29px;
+  padding-top: 4px;
+}
+.tips__link {
+  display: block;
+  position: relative;
+  width: 335px;
+}
+.tips__link::before, .tips__link::after {
+  content: "";
+  width: 10px;
+  height: 2px;
+  background-color: #37e63e;
+  position: absolute;
+  top: 25px;
+}
+.tips__link::before {
+  transform: rotate(90deg);
+  right: 14px;
+}
+.tips__link::after {
+  transform: rotate(-4deg);
```

Comment:
I think there are no such styles in design.

Reply:
fixed a bug